src/quantum/azext_quantum/tests/latest/test_quantum_jobs.py

```diff
@@ -0,0 +1,81 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+
+import os
+import unittest
+
+from azure_devtools.scenario_tests import AllowLargeResponse
+from azure.cli.testsdk import (ScenarioTest, ResourceGroupPreparer)
+
+from .utils import is_private_preview_subscription, TEST_WORKSPACE, TEST_RG, TEST_SUBS
+from ..._client_factory import _get_data_credentials
+from ...operations.workspace import WorkspaceInfo
+from ...operations.target import TargetInfo
+from ...operations.job import _generate_submit_args, _parse_blob_url
+
+TEST_DIR = os.path.abspath(os.path.join(os.path.abspath(__file__), '..'))
+
+
+class QuantumScenarioTest(ScenarioTest):
+
+    def test_targets(self):
+        # Since azure quantum is still in private preview, we require
+        # these tests to run in a specific subscription (AzureQuantum-test)
+        # if running somewhere else, just skip
+        if not is_private_preview_subscription(self):
+            return
```

Comment:
I'm not familiar with unit tests in Azure CLI, but is there something like a `skip_if` decorator that can be used to mark this test as skipped rather than passed?